package.json

```diff
@@ -21,10 +21,18 @@
                 "language": "javascript",
                 "path": "./snippets/snippets.json"
             },
-			{
-				"language": "javascriptreact",
-				"path": "./snippets/snippets.json"
-			}
+            {
+                "language": "javascriptreact",
+                "path": "./snippets/snippets.json"
+            },
+            {
+                "language": "typescript",
+                "path": "./snippets/snippets.json"
+            },
+            {
+                "language": "typescriptreact",
```

Comment:
I think this will not work

Reply:
Test and on vscode and try again because the javascriptreact is an extension

Reply:
`javascriptreact` refers to `.jsx` files, `typescriptreact` refers to `.tsx` files. What did you mean by "is an extension"?

Reply:
i will try here on my vscode

Reply:
Why this not work? It's from https://code.visualstudio.com/docs/languages/identifiers

Reply:
Just make it support .ts and .tsx